build-missing-docs.sh

```diff
@@ -31,10 +31,10 @@ TMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'tmp.XXXXXX')
 
 
 # Who we are for git
-git config --global user.email "main@ponylang.io"
-git config --global user.name "ponylang-main"
+#git config --global user.email "main@ponylang.io"
+#git config --global user.name "ponylang-main"
 # Other git setup
-git config --global push.default simple
+#git config --global push.default simple
```

Comment:
is there a reason these were commented out @ChristopherBiscardi ?

should the commenting out be reverted?